include/plog/Appenders/ConsoleAppender.h

```diff
@@ -18,6 +18,8 @@ namespace plog
                 m_stdoutHandle = GetStdHandle(stdHandle::kOutput);
             }
         }
+#elif __rtems__
+        ConsoleAppender() : m_isatty(true) {}

```

Comment:
Why not using `isatty` check?

Reply:
I think if you compile rtems without the `--enable-posix` option then `isatty` is not available. At least I got an undefined reference. I will check when I am back in the office.

Reply:
It worked fine without `--enable-posix`.

Reply:
@thesummer Could you check that on your setup?

Reply:
Ok, I investigated a bit further. It appears that in the current setup we have (rtems4.10) the fileno function is not declared in stdio.h. 
If I replace `m_isatty(!!isatty(fileno(stdout)))` with `m_isatty(!!isatty(STDOUT_FILENO))` everything works fine.
`STDOUT_FILENO` is defined in `unistd.h`

I will have to check if we have a system with rtems4.11 running somewhere.

Reply:
I checked with another setup with rtems4.12 with the `fileno` version. Everything builds fine.

Reply:
I tested on 4.11 and it worked with `fileno`.

Reply:
Ok. I removed the change here. We will phase out rtems4.10 soon hopefully and I can have the change locally for the time being.

I also and replaced the `#elif __rtems__` with `#elif defined(__rtems__)` as I was getting more undef warnings in Linux with that.